### PR TITLE
Add support for Aerotenna OcPoC-Zynq Mini hardware

### PR DESCRIPTION
**Pull request by davidaroyer**
Adds necessary changes to support Aerotenna OcPoC-Zynq Mini flight controller hardware.

### CODE REVIEW

**Comment by LorenzMeier** [comment]
Awesome! Can you contact me at lorenz@px4.io to discuss how we can test hardware to cross-validate this? Thanks!

File: src/drivers/drv_led.h [hunk 1]
```diff
@@ -57,5 +57,8 @@
 // set the queue size to the number of LED's, so that each led can be controlled individually
 static const int LED_UORB_QUEUE_LENGTH = BOARD_MAX_LEDS;
 
-
+#if defined (__PX4_POSIX_OCPOC)
```
Comment (davids5) [inline src/drivers/drv_led.h]: In the case of board specific changes. So that they can be reused by other boards, they should come from the board_config.h 
So in  board_config.h  add:
```#define BOARD_RGBLED0_PATH "/dev/i2c-1"```

Then here:
```
#if defined(BOARD_RGBLED0_PATH)
#  define RGBLED0_DEVICE_PATH BOARD_RGBLED0_PATH
#else
 #  define RGBLED0_DEVICE_PATH "/dev/rgbled0"
#endif
```

File: src/drivers/ocpoc_sbus_rc_in/CMakeLists.txt [hunk 2]
```diff
@@ -0,0 +1,43 @@
+############################################################################
```
Comment (davids5) [inline src/drivers/ocpoc_sbus_rc_in/CMakeLists.txt]: As a general question for the additions here: Would this driver be reusable as a general driver? If so we should remove the ocpos_ prefix.

Comment (davidaroyer) [inline src/drivers/ocpoc_sbus_rc_in/CMakeLists.txt]: @davids5 I imagine it's probably generic enough to be used as a general driver, but I don't really have the capacity to test that out. I would hate for someone down the line to assume it works when it may not, thus producing some major headaches.

File: src/lib/rc/sbus.cpp [hunk 3]
```diff
@@ -231,6 +286,32 @@ bool
 sbus_input(int sbus_fd, uint16_t *values, uint16_t *num_values, bool *sbus_failsafe, bool *sbus_frame_drop,
 	   uint16_t max_channels)
 {
+#if defined(__PX4_POSIX_OCPOC)
```
Comment (davidaroyer) [inline src/lib/rc/sbus.cpp]: @bkueng I know we've had issues before with normal rc input drivers due to some differences in our hardware. This particular part was put together by a colleague, specifically for our flight controller. I'll try reverting back to the original function here and get back to you soon.

File: posix-configs/ocpoc/px4.config [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+sleep 1
+param set MAV_TYPE 2
+param set MAV_SYS_ID 1
+param set BAT_CNT_V_VOLT 18.857142857
+param set BAT_V_DIV 0.000244141
+
+df_mpu9250_wrapper start_without_mag -R 10
+df_hmc5883_wrapper start
+df_ms5611_wrapper start
+rgbled start -b /dev/i2c-1
```
Comment (davidaroyer) [inline posix-configs/ocpoc/px4.config]: I think I remember seeing the same thing, that this doesn't actually do anything, and I probably just overlooked taking this -b option out from the px4.config file.

File: src/lib/rc/sbus.cpp (hunk 3 above)
Comment (davidaroyer) [inline src/lib/rc/sbus.cpp]: Alright, it looked great in flight, no issues with control. Thanks for pointing this out; it seems that's been an unnecessary change all along.

**Comment by davidaroyer** [comment]
@bkueng I've now moved the battery monitor to a separate driver. I'll get to the documentation as soon as I can.
